### ISSUE TITLE
McRetroTS error

### ISSUE THREAD

**Issue by colinhawco** [josephdviviano/epitome#57]
The following line doesn't run in my script:

 ${McRetroTS} \
                    ${runfolder}/resp.*.phys ${runfolder}/card.*.phys ${tr} ${slice} ${fs} 0

Error:
/scratch/colin/epitome//tmsfmri_study1_2017/P103/TMS/SESS01//RUN01//resp.04.phys: line 1: 6.6375732421875: command not found

I believe the variable  ${McRetroTS} is unset, I expect its a hold over from an older version where this function was not implemented as a built in module?

**Comment by josephdviviano** [comment]
Why is this line running? Do you have .phys files?

**Comment by josephdviviano** [comment]
If so you need to configure McRetroTS as per the installation instructions.

**Comment by colinhawco** [comment]
yeah I have phys files, and I want to include it. 

I foudn the bit in the readme I missed. 

"For physiological noise regression, you must have the MATLAB compiler runtime installed, along with AFNI's McRetroTS scripts installed in /opt/MATLAB/MATLAB_Compiler_Runtime/ and /opt/mcretro/, respectively." 

but to my knowledge I have no power to install anything in /opt/.  The matlab module is loaded. No idea how to get mcretro into any kind of path. It doesn't appear to be a module.

**Comment by josephdviviano** [comment]
You can install mcretro yourself in your projects folder: https://sscc.nimh.nih.gov/sscc/dglen/McRetroTS

**Comment by colinhawco** [comment]
I think I already have it:

export PATH='/scratch/colin/MCR/v715/bin':$PATH
export LD_LIBRARY_PATH='/scratch/colin/MCR/v715/runtime/glnxa64':$LD_LIBRARY_PAT

This worked in my previous versions of epitome. 

 ${McRetroTS} still seems to be a non-existent variable though. It never gets set.

**Comment by colinhawco** [comment]
I got it to work. I had to specify the variable ${McRetroTS}  as 

McRetroTS=/projects/colin/applications/epitome/bin/run_McRetroTS.sh

And add an input argument, pathing to my library file as needed by the SH file. 
/scratch/colin/MCR/v715/

So my code now reads:

McRetroTS=/projects/colin/applications/epitome/bin/run_McRetroTS.sh

${McRetroTS} /scratch/colin/MCR/v715/ ${runfolder}/resp..phys ${runfolder}/card..phys ${tr} ${slice} ${fs} 0

I think the current version of the physio is bugged. McRetroTS seems to be included in the epitome version, but the generated code needs to be updated. The current readme doesn't seem to really cover this (the challenges of documentation, on which I am in a position to criticize exactly nobody!)

**Comment by josephdviviano** [comment]
Yes sorry the documentation is really out of date, and the mcretrots thing isn't used by anyone other than yourself at the minute. I should will fit (I'll leave the issue open until it is fixed).

**Comment by josephdviviano** [comment]
Hi Colin,

I now include all of MCR v7.15 in epitome, and have fixed `init_basic` to automatically set it up for you. Hopefully it works going forward without you having to mess around with the scripts!

**Comment by josephdviviano** [comment]
nevermind -- mcretrots takes up too much space in git LFS -- but I'm pinging the AFNI folks to make sure they host a copy of the file somehwere.


Perhaps I'll need to transition to another method for physio noise regression.

**Comment by josephdviviano** [comment]
Great news -- the re-wrote mcretrots in python so we don't need to bother with this MCR thing any longer.

**Comment by josephdviviano** [comment]
I'll update the module to use `RetroTS.py`
